plugins/lang-javascript/src/main/java/org/elasticsearch/script/javascript/JavaScriptScriptEngineService.java

```diff
@@ -172,14 +172,16 @@ public Object compile(String scriptName, String scriptSource, Map<String, String
     }
 
     @Override
-    public ExecutableScript executable(CompiledScript compiledScript, Map<String, Object> vars) {
+    public ExecutableScript executable(CompiledScript compiledScript, @Nullable Map<String, Object> vars) {
```

Comment:
This is outside the scope of the PR, but we should look into requiring vars to be non null in the future